jadx-core/src/main/java/jadx/core/utils/files/InputFile.java

```diff
@@ -134,7 +135,7 @@ private boolean loadFromZip(String ext) throws IOException, DecodeException {
 							case ".jar":
 								index++;
 								Path jarFile = Files.createTempFile(entryName, ".jar");
```

Comment:
I just notice that in previous commits you replace many calls from `FileUtils.createTempFile` to `Files.createTempFile` and now temp files not deleted on exit and pollute temp dir, because `FileUtils.createTempFile`  invoke `file.deleteOnExit();` to remove temp file, but now it is not :(
Also names was nice like starting with `jadx-tmp-` so it easier to understard which app created it:
```
before: /tmp/jadx-tmp-8207354968595972366-classes.jar
after: /tmp/classes.jar8207354968595972366.jar
```
So, please, next time try to not change current behavior heavily, because sometimes it is hard to notice such changes on review :((

Reply:
You are right, I have a bad habit of mixing multiple changes.

I thought using the new API would be better.

A new PR is underway.